engine/classes/ElggPluginPackage.php

```diff
@@ -297,7 +300,7 @@ private function hasSaneDependencies() {
 				foreach (${$dep_type} as $dep) {
 					if (!in_array($dep['type'], $this->depsSupportedTypes)) {
 						$this->errorMsg =
-							_elgg_services()->translator->translate('ElggPluginPackage:InvalidPlugin:InvalidDependency', [$dep['type']]);
+							elgg_echo('ElggPluginPackage:InvalidPlugin:InvalidDependency', [$dep['type']]);
```

Comment:
Is it possible to keep the `elgg_echo` on the same line a `$this->errorMsg`? now that the line is shorter.....